File: shoring/blackjackEngine/src/test/java/com/panopset/tests/blackjackEngine/SessionData.java
package com.panopset.tests.blackjackEngine;

interface SessionData {
 
 String[] BLACKJACK_WWCD = new String[] {
  "ace of clubs",
  "six of clubs",
  "jack of clubs",
  "jack of clubs",
  "jack of clubs"
 };
 
 String[] SPLITACES_DLRBLACKJACK = new String[] {
  "ace of clubs",
  "ace of clubs",
  "ace of clubs",
  "jack of clubs",
  "jack of spades",
  "jack of hearts",
  "jack of diamonds"
 };
 
 String[] SPLITACES_DLRHIT_TO_21 = new String[] {
  // initial deal
  "ace of clubs",
  "four of clubs",
  "ace of clubs",
  "jack of clubs",
  
  // player splits to to soft 21s
  "jack of spades",
  "jack of hearts",
  
  // dealer hits his 14 to 21.
  "seven of diamonds"
 };
 
 String[] SPLITACES_DLRTIE = new String[] {
  "ace of clubs",
  "ace of clubs",
  "ace of clubs",
  "jack of clubs",
  "jack of clubs",
  "jack of spades",
 };
}

File: code/shoring/blackjackEngine/src/test/kotlin/com/panopset/tests/blackjackEngine/SessionData.kt
package com.panopset.tests.blackjackEngine

val BLACKJACK_WWCD = arrayOf(
    "ace of clubs",
    "six of clubs",
    "jack of clubs",
    "jack of clubs",
    "jack of clubs"
)
val SPLITACES_DLRBLACKJACK = arrayOf(
    "ace of clubs",
    "ace of clubs",
    "ace of clubs",
    "jack of clubs",
    "jack of spades",
    "jack of hearts",
    "jack of diamonds"
)
val SPLITACES_FIRST_HAND_PLAYER_BLACKJACK = arrayOf(
    "ace of clubs",
    "seven of spades",
    "ace of hearts",
    "king of spades",
    "ten of diamonds",
    "eight of spades"
)
val SPLITACES_DLRHIT_TO_21 = arrayOf( // initial deal
    "ace of clubs",
    "four of clubs",
    "ace of clubs",
    "jack of clubs",  // player splits to soft 21s
    "jack of spades",
    "jack of hearts",  // dealer hits his 14 to 21.
    "seven of diamonds"
)
val SPLITACES_DLRTIE = arrayOf(
    "ace of clubs",
    "ace of clubs",
    "ace of clubs",
    "jack of clubs",
    "jack of clubs",
    "jack of spades"
)
